api/institutions/serializers.py

```diff
@@ -40,3 +47,49 @@ def get_absolute_url(self, obj):
 
     class Meta:
         type_ = 'institutions'
+
+
+class NodeRelated(JSONAPIRelationshipSerializer):
+    id = ser.CharField(source='_id', required=False, allow_null=True)
+    class Meta:
+        type_ = 'nodes'
+
+class InstitutionNodesRelationshipSerializer(ser.Serializer):
+    data = ser.ListField(child=NodeRelated())
+    links = LinksField({'self': 'get_self_url',
+                        'html': 'get_related_url'})
+
+    def get_self_url(self, obj):
+        return obj['self'].nodes_relationship_url
+
+    def get_related_url(self, obj):
+        return obj['self'].nodes_url
+
+    class Meta:
+        type_ = 'nodes'
+
+    def create(self, validated_data):
+        inst = self.context['view'].get_object()['self']
+        user = self.context['request'].user
+        node_dicts = validated_data['data']
+        nodes = [Node.load(node_dict['_id']) for node_dict in node_dicts]
+
+        changes_flag = False
+        for node in nodes:
+            if not node:
+                raise exceptions.NotFound
+            if not node.has_permission(user, osf_permissions.ADMIN):
+                raise exceptions.PermissionDenied
+
+        for node in nodes:
```

Comment:
Because wanted behavior is to not update any node if the request has any problems with it.